### PR TITLE
Home work #19. Fixtures

### PR DESCRIPTION
**Pull request by AndreyTests**

### CODE REVIEW

File: src/fixtures/test.fixtures.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import GaragePage from '../pages/GaragePage';
+import {STORAGE_STATE_USER_PATH} from "../data/storageState";
+
+export const test = base.extend(({
+    garagePage: async ({browser}, use)=>{
+        const browserContext = await browser.newContext({
+            storageState: STORAGE_STATE_USER_PATH
+        });
+        const page = await browserContext.newPage();
+        const garagePage = new GaragePage(page);
```
Comment (sttaran) [inline src/fixtures/test.fixtures.js:11]: it is supposed you do navigation right here

File: src/pages/GaragePage.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+            page.locator('app-panel-layout' , {has : page.locator('button', {hasText: 'Add car'})}));
+    }
+
+    async getMainTitle() {
+        await this._page.title();
+    }
+
+    async getPageUrl() {
+        await this._page.url();
+    }
```
Comment (sttaran) [inline src/pages/GaragePage.js:15]: all these methods should be in BasePage

Comment (sttaran) [reply src/pages/GaragePage.js:15]: but I recommend to use page element for assertion, for example "Add car" button

**Review by sttaran — CHANGES_REQUESTED** [review]

File: src/pages/GaragePage.js (hunk 2 above)
Comment (AndreyTests) [reply src/pages/GaragePage.js:15]: I'll fixed all comments in next pull.

**Pull request merged**